mackup/applications/rofi.cfg

```diff
@@ -0,0 +1,5 @@
+[application]
+name = rofi
+
+[configuration_files]
+.config/rofi/config.rasi
```

Comment:
You should set your edit to add a newline char at the end of each line, I'll fix this one.